code/datums/jobs/job/shipside.dm

```diff
@@ -743,7 +755,7 @@ Use your office fax machine to communicate with corporate headquarters or to acq
 		/datum/job/terragov/squad/specialist = SPEC_POINTS_REGULAR,
 		/datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR,
 	)
-
+    job_points_needed = 40
```

Comment:
there's an issue with identation here (too much space).

Please don't webeddit, use DM or VSC

Reply:
alright, I will do it later, thanks man.